### PR TITLE
HDDS-11562. Parameterize TestSCMNodeManager#testProcessLayoutVersion

### PR DESCRIPTION
**Pull request by weimingdiit**
## What changes were proposed in this pull request?
Refactor testProcessLayoutVersion  method to use org.junit.jupiter so test parameterization can be used

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-11562

## How was this patch tested?
unit tests

### CODE REVIEW

File: hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/node/TestSCMNodeManager.java [hunk 1]
```diff
@@ -17,53 +17,40 @@
...
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.List;
-import java.util.Set;
-import java.util.UUID;
-import java.util.concurrent.ScheduledFuture;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.TimeoutException;
-
```
Comment (adoroszlai) [inline hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/node/TestSCMNodeManager.java:32]: Please don't move around imports unnecessarily.  It inflates the patch and makes harder to review.  It also increases probability of conflict for backports.

**Review by adoroszlai — COMMENTED** [review]
Thanks @weimingdiit for the patch.

File: hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/node/TestSCMNodeManager.java (hunk 1 above)
Comment (weimingdiit) [reply hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/node/TestSCMNodeManager.java]: Thank you for your comment, I have updated the code, please review it again

**Review by adoroszlai — APPROVED** [review]

**Comment by slfan1989** [comment]
LGTM +1.

**Review by slfan1989 — APPROVED** [review]

**Comment by adoroszlai** [comment]
Thanks @weimingdiit for the patch, @slfan1989 for the review.

**Comment by weimingdiit** [comment]
@slfan1989 Thanks for your review and @adoroszlai  Thanks for your merge

**Pull request merged**